### PR TITLE
Add 120 foot reach to NPC strike list

### PR DESCRIPTION
**Pull request by TikaelSol**
Needed for Battlezoo but also both of these things are ones I can see Paizo winding up at eventually anyways.

### CODE REVIEW

**Comment by stwlam** [comment]
having trouble making sense of a "visual" strike

File: src/scripts/config/traits.ts [hunk 1]
```diff
@@ -579,12 +579,14 @@ const npcAttackTraits = {
...
     "reach-60": "PF2E.TraitReach60",
     "reach-100": "PF2E.TraitReach100",
+    "reach-120": "PF2E.TraitReach120",
     "reach-200": "PF2E.TraitReach200",
     "reach-1000": "PF2E.TraitReach1000",
     "reload-0": "PF2E.TraitReload0",
     "reload-1": "PF2E.TraitReload1",
     "reload-2": "PF2E.TraitReload2",
     "reload-1-min": "PF2E.TraitReload1Min",
+    visual: "PF2E.TraitVisual"
```
Comment (TikaelSol) [inline src/scripts/config/traits.ts:589]:
```suggestion
    visual: "PF2E.TraitVisual",
```

**Comment by TikaelSol** [comment]
The thing throws a twisted reflection of you to deal mental damage.

File: src/scripts/config/traits.ts [hunk 2]
```diff
@@ -579,12 +579,14 @@ const npcAttackTraits = {
...
     "reach-60": "PF2E.TraitReach60",
     "reach-100": "PF2E.TraitReach100",
+    "reach-120": "PF2E.TraitReach120",
     "reach-200": "PF2E.TraitReach200",
     "reach-1000": "PF2E.TraitReach1000",
     "reload-0": "PF2E.TraitReload0",
     "reload-1": "PF2E.TraitReload1",
     "reload-2": "PF2E.TraitReload2",
     "reload-1-min": "PF2E.TraitReload1Min",
+    visual: "PF2E.TraitVisual",
```
Comment (TikaelSol) [inline src/scripts/config/traits.ts:589]:
```suggestion
```

**Pull request merged**